README.md

```diff
@@ -312,13 +312,64 @@ There's several different mechanisms you can use to define custom remappings. Al
     ]
 ```
 
-#### `"vim.insertModeKeyBindingsNonRecursive"`/`"otherModesKeyBindingsNonRecursive"`
+* Bind `p` in visual mode to paste without overriding the current register
 
-* Non-recursive keybinding overrides to use for insert and other (non-insert) modes (similar to `:noremap`)
-* *Example:* Bind `j` to `gj`. Notice that if you attempted this binding normally, the j in gj would be expanded into gj, on and on forever. Stop this recursive expansion using insertModeKeyBindingsNonRecursive and/or otherModesKeyBindingNonRecursive.
+```json
+    "vim.visualModeKeyBindingsNonRecursive": [
+        {
+            "after": [
+                "\"",
+                "_",
+                "d",
+                "P"
+            ],
+            "before": [
+                "p",
+            ]
+        }
+    ],
+```
+
+* Bind `>` and `<` in visual mode to indent/outdent lines (repeatable)
+
+
+```json
+    "vim.visualModeKeyBindingsNonRecursive": [
+        {
+            "after": [],
```

Comment:
Can you remove all the empty arguments?

Reply:
@jpoon done https://github.com/VSCodeVim/Vim/pull/2726/commits/65c24f313f41d60d9a685bf4f205d967dd673dd6